tests/js/unit/models/veventModelSpec.js

```diff
@@ -2050,4 +2050,90 @@ END:VEVENT`.split("\n").join("\r\n"));
 
 		expect(called).toEqual(true);
 	});
+
+	var using = function (values, func) {
```

Comment:
@PVince81 this is some data provider code I was taking from the npm package jasmine-data-provider.
I could not get it to work properly ....
Is it worth putting this some there else? THX